### ISSUE TITLE
What is the difference between ROS2 plugin and FastRTPS plugin?

### ISSUE THREAD

**Issue by chaoli2** [ApexAI/performance_test#77]
We notice that there are three provided plugins (ROS2, FastRTPS and ConnextDDSMicro). But after we test ROS2 and FastRTPS, we find very minor differences between them, like below screenshot.
![image](https://user-images.githubusercontent.com/24284449/62098236-22dcc400-b2bc-11e9-93c3-c355ab0a2d37.png)

![image](https://user-images.githubusercontent.com/24284449/62098276-3851ee00-b2bc-11e9-971b-b4ddf460b49e.png)

Intuitively, FastRTPS plugin might contain only DDS latency while ROS2 plugin might contain DDS + RMW latency. If so, there should be obvious difference between them. Not sure that whether this is the right cognition of provided plugins.

**Comment by deeplearningrobotics** [comment]
@chaoli2: Yes, the FastRTPS plugin does not use ROS 2 at all. It uses the following code here: https://github.com/ApexAI/performance_test/blob/master/performance_test/src/communication_abstractions/fast_rtps_communicator.hpp

If you suspect that because of some bug the wrong plugin is selected you can use GDB or perf to ensure that you are running the code you are expected to be run.
